test/autorest/generated/paginggroup/paging.go

```diff
@@ -46,6 +46,10 @@ type PagingOperations interface {
 	GetSinglePages() (ProductResultPager, error)
 	// GetSinglePagesFailure - A paging operation that receives a 400 on the first call
 	GetSinglePagesFailure() (ProductResultPager, error)
+	// GetWithQueryParams - A paging operation that includes a next operation. It has a different query parameter from it's next operation nextOperationWithQueryParams. Returns a ProductResult
+	GetWithQueryParams(requiredQueryParameter int32) (ProductResultPager, error)
+	// NextOperationWithQueryParams - Next operation for getWithQueryParams. Pass in next=True to pass test. Returns a ProductResult
+	NextOperationWithQueryParams(ctx context.Context) (*ProductResultResponse, error)
```

Comment:
This should be omitted from the interface as we call this internally.  There should already be code to do this, curious why this case was missed.

Reply:
NextOperationWithQueryParams is declared as its own operation in the swagger. Is there some field or condition that should be checked to omit this?

Reply:
In `generateOperation()` the first few lines should omit this (it does for other Next* methods).

Reply:
Ok, will see why this is being skipped then